### PR TITLE
[DOCFIX] Update Running-Alluxio-Locally.md

### PR DESCRIPTION
**Pull request by apc999**
For users running on MacOS, sudo privileges are not required as mount in MacOS doesn't require sudo

### CODE REVIEW

File: docs/en/Running-Alluxio-Locally.md [hunk 1]
```diff
@@ -34,13 +34,23 @@ repeatedly input the password, you can add the public ssh key for the host into
...
 
-## Start Alluxio with sudo privileges
-
-By default, on startup Alluxio will create a
+By default, Alluxio Filesystem will create and use
 [RAMFS](https://www.kernel.org/doc/Documentation/filesystems/ramfs-rootfs-initramfs.txt) as its in-memory data storage.
-This step requires sudo privileges to perform "mount", "umount", "mkdir" and "chmod" operations. There are two approaches to achieve this:
+Depends on the operating systems and environment, it may require different privileges to complete this step.
+
+## Running on MacOS
```
Comment (jsimsa) [inline docs/en/Running-Alluxio-Locally.md]: I think both MacOS and Linux require sudo, so I am not sure why are you distinguishing between the two.

What I would do instead is:

```
...

To provide memory-speed I/O, Alluxio stores data in a RAMdisk. When starting Alluxio, you can either instruct Alluxio to use an existing RAMdisk or to create the RAMdisk on your behalf (NOTE: this step requires sudo privileges).

### Starting Alluxio with Existing RAMdisk
...

### Starting Alluxio without Existing RAMdisk
...
```

Comment (apc999) [inline docs/en/Running-Alluxio-Locally.md]: @jsimsa  if you check `mount_ramfs_mac()` in `bin/alluxio-mount.sh`, 
on mac we actually never use sudo to mount.

Comment (apc999) [inline docs/en/Running-Alluxio-Locally.md]: and using sudo to start `alluxio` creates problems actually.

Comment (jsimsa) [inline docs/en/Running-Alluxio-Locally.md]: as discussed offline, let's reorganize this to shift the focus from `sudo` onto how to start Alluxio in the common case (where it just works)

**Comment by apc999** [comment]
@jsimsa changed made. PTAL

File: docs/en/Running-Alluxio-Locally.md [hunk 2]
```diff
@@ -25,72 +28,73 @@ filesystem (e.g., `alluxio.underfs.address=/tmp`).
...
-With the proper user, run the following command to start Alluxio filesystem.
+Simply run the following command to start Alluxio filesystem.
 
 ```bash
 $ ./bin/alluxio-start.sh local
 ```
+> NOTE: On Linux, this command may require to input password to get sudo privileges 
+> in order to setup RAMFS. If you do not want to type in the password every time, or you do 
+> not even have sudo privileges, please read the alternative approaches in [FQA](#faq).
+
```
Comment (jsimsa) [inline docs/en/Running-Alluxio-Locally.md]: remove extra line

File: docs/en/Running-Alluxio-Locally.md [hunk 3]
```diff
@@ -25,72 +28,73 @@ filesystem (e.g., `alluxio.underfs.address=/tmp`).
...
-Alternatively, if a RAMFS (e.g., `/path/to/ramdisk`) is already mounted by the system admin, you can specify the path in
+# FAQ
+
+## Why is sudo privilege needed to start Alluxio on Linux?
+
+By default, Alluxio filesystem uses [RAMFS](https://www.kernel.org/doc/Documentation/filesystems/ramfs-rootfs-initramfs.txt) as its in-memory data storage. It turns out on MacOS, it is fine for Alluxio to mount a RAMFS without being a super user. However, on Linux, it requires sudo privileges to perform "mount" (and the followed "umount", "mkdir" and "chmod" operations). 
+
+## Can I still try Alluxio on Linux without sudo privileges?
+
+If you have no sudo privileges on Linux, for Alluxio Filesystem to work, it requires a RAMFS (e.g., `/path/to/ramdisk`) already mounted by the system admin and accessible for read/write-operations by the user. In this case you can specify the path in
```
Comment (yupeng9) [inline docs/en/Running-Alluxio-Locally.md]: Shouldn't we mention the option of using tmpfs?

File: docs/en/Running-Alluxio-Locally.md (hunk 2 above)
Comment (apc999) [inline docs/en/Running-Alluxio-Locally.md]: done

**Comment by apc999** [comment]
@jsimsa @yupeng9 comments addressed

File: docs/en/Running-Alluxio-Locally.md [hunk 4]
```diff
@@ -73,20 +73,24 @@ By default, Alluxio filesystem uses [RAMFS](https://www.kernel.org/doc/Documenta
 
 ## Can I still try Alluxio on Linux without sudo privileges?
 
-If you have no sudo privileges on Linux, for Alluxio Filesystem to work, it requires a RAMFS (e.g., `/path/to/ramdisk`) already mounted by the system admin and accessible for read/write-operations by the user. In this case you can specify the path in
+If you have no sudo privileges on Linux, for Alluxio Filesystem to work, it requires a RAMFS (e.g., `/path/to/ramdisk`) already mounted
+by the system admin and accessible for read/write-operations by the user. Alternatively, you can also specify Linux [tmpFS](https://en.wikipedia.org/wiki/Tmpfs)
```
Comment (yupeng9) [inline docs/en/Running-Alluxio-Locally.md]: max width per line is 100 characters

File: docs/en/Running-Alluxio-Locally.md [hunk 5]
```diff
@@ -73,20 +73,24 @@ By default, Alluxio filesystem uses [RAMFS](https://www.kernel.org/doc/Documenta
 
 ## Can I still try Alluxio on Linux without sudo privileges?
 
-If you have no sudo privileges on Linux, for Alluxio Filesystem to work, it requires a RAMFS (e.g., `/path/to/ramdisk`) already mounted by the system admin and accessible for read/write-operations by the user. In this case you can specify the path in
+If you have no sudo privileges on Linux, for Alluxio Filesystem to work, it requires a RAMFS (e.g., `/path/to/ramdisk`) already mounted
+by the system admin and accessible for read/write-operations by the user. Alternatively, you can also specify Linux [tmpFS](https://en.wikipedia.org/wiki/Tmpfs)  
+as the data storage. Tmpfs is a temporary file storage backed by memory (e.g., typically `/dev/shm` on Linux), but may use SWAP space and
+therefore provides less performance guarantees compared to ramfs. In this case you have can specify the path in
```
Comment (yupeng9) [inline docs/en/Running-Alluxio-Locally.md]: It's not very straightforward to understand that the path `/path/to/ramdisk` can also be the path to tmpfs.

I suggest you move the paragraph of tmpFS after the code instruction, and explicitly call out the change on ` alluxio.worker.tieredstore.level0.dirs.path=/dev/shm`

Comment (apc999) [inline docs/en/Running-Alluxio-Locally.md]: @yupeng9  updated

**Pull request merged**